codeHF/dpl-config_run5_hf.json

```diff
@@ -643,6 +687,28 @@
         },
        "processMC": "true"
     },
+    "hf-task-lb": {
+        "cutYCandMax": "2.0",
+        "d_selectionFlagLb": "0",
```

Comment:
```suggestion
        "selectionFlagLb": "0",
```

Reply:
Is there any particular reason why some use the d_ prefix and some don't?

Reply:
The first tasks used the prefix and some other tasks took it from there. It has been fixed somewhere but not everywhere yet. The conventional way is without the prefix.